include/game/game.h

```diff
@@ -312,16 +343,16 @@ typedef struct Player_ {
     /* 0x85 */ s8 character;
     /* 0x86 */ u8 unk86;
     /* 0x87 */ u8 unk87;
-    /* 0x88 */ u8 filler88[4];
+    /* 0x88 */ u8 unk88;
+    /* 0x88 */ u8 filler88[3];
     /* 0x8C */ struct Task *spriteTask;
     /* 0x90 */ UNK_3005A70 *unk90;
 
     // TODO: Only used for Cream/Tails?
     //       Alternatively, some of the following data might be a union
     /* 0x94 */ UNK_3005A70 *unk94;
     /* 0x98 */ u8 unk98; // Multiplayer var. TODO: check sign!
```

Comment:
It's it more likely `unk98` and `unk99` are combined? Especially as the InitPlayer function function memsets unk98 too?